styles/community-styles.css

```diff
@@ -14,6 +15,7 @@ main.liveby-community .hero-container {
   margin-top: 0;
   padding-left: 0;
   padding-right: 0;
+  max-width: 1600px;
```

Comment:
let's move it to the variable. I'm using this value in my styles as well.

Reply:
We'll need two. Max width of hero is 1600, max width of content (including nav) is 1350